scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -80,108 +79,112 @@
 y.value_counts(True)
```

Comment:
I had some other comments as well, nothing too substantial, but I'm not seeing them anywhere in this thread... not sure what may have happened. @patrick-miller if you don't see any of my other comments, let me know and I'll re-add them tomorrow morning.